crawl-ref/source/cloud.cc

```diff
@@ -1383,10 +1303,27 @@ int actor_apply_cloud(actor *act)
         act->hurt(oppressor, final_damage, BEAM_MISSILE,
                   KILLED_BY_CLOUD, "", cloud.cloud_name(true));
     }
+}
 
-    return final_damage;
+// Applies damage and side effects for an actor in a cloud and returns
+// the damage dealt.
+void actor_apply_cloud(actor *act)
+{
+    cloud_struct* cl = cloud_at(act->pos());
+    if (!cl)
+        return;
+
+    cloud_struct cloud = *cl;
+
+    if (cloud.type == CLOUD_STORM){
```

Comment:
Minor: we use [Allman](https://en.wikipedia.org/wiki/Indentation_style#Allman_style) as our house style. Braces on the next line. (Not my preference, but we gotta have a standard; otherwise it's chaos!)